CHANGELOG.md

```diff
@@ -12,9 +12,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - test for biceps:5-4-7_14
 - test for biceps:5-4-7_16
 - spotless plugin
+- test for glue:8-1-3
 
 ### Changed
 - checkstyle plugin scope via checkstyle.xml
+- updated test case for glue:R0036 to glue:R0036_0
```

Comment:
not part of this pr

Reply:
AFAIK, this was merged in. However, it is not present on the main branch and is hence shown as an addition.
No idea how this got in here. Will remove it here and hope that this does not lead to unexpected results later.